### PR TITLE
photo album repository and UI updated

### PR DESCRIPTION
**Pull request by hojunchoi999**

### CODE REVIEW

File: app/src/main/java/com/example/packpals/models/PhotoAlbum.kt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package com.example.packpals.models
+
+data class PhotoAlbum (
+    var albumId: String,
+    val title: String,
+    val coverPhotoUrl: Int,
```
Comment (stephchi0) [inline app/src/main/java/com/example/packpals/models/PhotoAlbum.kt]: why is this an int

File: app/src/main/java/com/example/packpals/repositories/PhotoRepository.kt [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import com.google.firebase.firestore.CollectionReference
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.StorageReference
+
+import kotlinx.coroutines.tasks.await
+import javax.inject.Inject
+
+class PhotoRepository @Inject constructor (private val photoCollectionRef: CollectionReference) {
+
+    private val storage = FirebaseStorage.getInstance()
```
Comment (stephchi0) [inline app/src/main/java/com/example/packpals/repositories/PhotoRepository.kt:14]: probably just use StorageRepository for any firebase cloud storage calls and add any functions you need there instead

**Review by stephchi0 — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/packpals/models/PhotoAlbum.kt (hunk 1 above)
Comment (hojunchoi999) [reply app/src/main/java/com/example/packpals/models/PhotoAlbum.kt]: cuz its just for hard coded data rn. Its not a URL yet

File: app/src/main/java/com/example/packpals/repositories/PhotoRepository.kt (hunk 2 above)
Comment (hojunchoi999) [reply app/src/main/java/com/example/packpals/repositories/PhotoRepository.kt:14]: kk

File: app/src/main/java/com/example/packpals/models/PhotoAlbum.kt (hunk 1 above)
Comment (stephchi0) [reply app/src/main/java/com/example/packpals/models/PhotoAlbum.kt]: ok should probably be a string though no?

**Review by stephchi0 — APPROVED** [review]
good enough for now i guess lmao we'll merge after you fix conflicts so we can do the maps stuff

**Review by alex-fang0 — CHANGES_REQUESTED** [review]
Just wondering how photos might connect to itinerary items for displayin.

**Review by alex-fang0 — APPROVED** [review]
Could be done in another pr

**Pull request closed without merging**